typescript/sdk/src/ism/HyperlaneIsmFactory.ts

```diff
@@ -207,17 +281,24 @@ export class HyperlaneIsmFactory extends HyperlaneApp<ProxyFactoryFactories> {
     return IRoutingIsm__factory.connect(address, signer);
   }
 
-  private async deployAggregationIsm(
-    chain: ChainName,
-    config: AggregationIsmConfig,
-    origin?: ChainName,
-  ) {
+  private async deployAggregationIsm(params: {
```

Comment:
Is there a reason we make these methods private instead of protected? Seems unnecessary and short-sighted